src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -27,6 +35,14 @@ LRESULT CALLBACK HwndTerminal::HwndTerminalWndProc(
 
     if (terminal)
     {
+        if (_IsMouseMessage(uMsg) && terminal->_CanSendVTMouseInput())

```

Comment:
I really wish we could use `_IsMouseMessage` in the case statements, I don't like this being separate. But I also wouldn't want to see that large list of message types in the switch 😅

Reply:
> Don't like this being separate

I agree totally with all these points. Conhost has them all as case labels, and it's monstrous. I may come through afterwards and refactor this code for flow/order, so we shall get a chance to revisit :smile: